CIP-GuardScripts/README.md

```diff
@@ -0,0 +1,138 @@
+---
+CIP: ????
+Title: GuardScriptCredential and Guard Script Purpose
```

Comment:
```suggestion
Title: Guard Script Purpose and Credential
```
If this is technically inaccurate please @colll78 @Crypto2099 @fallen-icarus please suggest something both accurate _and_ concise.